tests/test_sqlite.py

```diff
@@ -0,0 +1,29 @@
+from ms2query.sqlite import create_table_structure, \
+    add_spectra_to_database, get_spectra_from_sqlite
+from ms2query.app_helpers import gather_test_json
+
+
+def test_sqlite_functions():
+    """Tests the functions create_table_structure, add_spectra_to_database
+    and get_spectra_from sqlite"""
+    column_type_dict = {"spectrum_id": "VARCHAR",
+                        "source_file": "VARCHAR",
+                        "ms_level": "INTEGER",
+                        "key_that_does_not_exist": "VARCHAR"
+                        }
+    sqlite_file_name = "test_spectra_database.sqlite"
```

Comment:
This will create an actual file when running.
Usually in a test it is nicer to just write to a temporary path/file. This can for instance be done as in https://github.com/matchms/matchms/blob/master/tests/test_save_as_json_load_from_json.py
(using the argument `tmp_path`)

Reply:
Is there a way of still test running the test locally, since now I need to pass an argument tmp_path to the function to run it. I tried pytest, but this gives an error that the argument tmp_path is not given.